### PR TITLE
Update Uphold

### PR DESCRIPTION
**Pull request by rmtsrc**

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -392,7 +392,9 @@ websites:
       sms: Yes
       phone: Yes
       software: Yes
-      doc: https://support.uphold.com/hc/en-us/articles/206119683-How-does-phone-number-verification-work-
+      exceptions:
+          text: "Software authentication requires the Authy app."
+      doc: https://support.uphold.com/hc/en-us/articles/210689043-How-to-set-up-Authy-Two-Factor-Authentication-for-Uphold
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion
      doc: https://support.uphold.com/hc/en-us/articles/210689043
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
According to [this page](https://support.uphold.com/hc/en-us/articles/360024042131-How-do-I-set-up-Two-Factor-Authentication-using-the-Uphold-mobile-app-) the site supports all TOTP apps using RFC-6238

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -392,7 +392,9 @@ websites:
       sms: Yes
       phone: Yes
       software: Yes
-      doc: https://support.uphold.com/hc/en-us/articles/206119683-How-does-phone-number-verification-work-
+      exceptions:
+          text: "Software authentication requires the Authy app."
+      doc: https://support.uphold.com/hc/en-us/articles/210689043
```
Comment (jamcat22) [inline _data/cryptocurrencies.yml]:
```suggestion
      doc: https://support.uphold.com/hc/en-us/articles/360024042471
```

File: _data/cryptocurrencies.yml [hunk 3]
```diff
@@ -392,7 +392,9 @@ websites:
       sms: Yes
       phone: Yes
       software: Yes
-      doc: https://support.uphold.com/hc/en-us/articles/206119683-How-does-phone-number-verification-work-
+      exceptions:
```
Comment (jamcat22) [inline _data/cryptocurrencies.yml]:
```suggestion
```

File: _data/cryptocurrencies.yml [hunk 4]
```diff
@@ -392,7 +392,9 @@ websites:
...
+          text: "Software authentication requires the Authy app."
```
Comment (jamcat22) [inline _data/cryptocurrencies.yml]:
```suggestion

```

File: _data/cryptocurrencies.yml [hunk 5]
```diff
@@ -392,7 +392,9 @@ websites:
       sms: Yes
```
Comment (jamcat22) [inline _data/cryptocurrencies.yml]:
```suggestion
```

File: _data/cryptocurrencies.yml [hunk 6]
```diff
@@ -392,7 +392,9 @@ websites:
...
       phone: Yes
```
Comment (jamcat22) [inline _data/cryptocurrencies.yml]:
```suggestion
```

**Review by jamcat22 — CHANGES_REQUESTED** [review]
Looks like Uphold just changed their 2FA to standard TOTP only, dropping support for Authy. They also changed their help center article ID, so I suggest changing the doc link and removing SMS, phone, and the exception text.

**Review by jamcat22 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**